recipes/miniz/all/conanfile.py

```diff
@@ -46,6 +46,8 @@ def _configure_cmake(self):
         return self._cmake
 
     def build(self):
+        for patch in self.conan_data["patches"][self.version]:
```

Comment:
```suggestion
        for patch in self.conan_data.get("patches", {}).get(self.version, []):
```
Next release will very likely not require a patch. This modification may avoid triggering a new build for the current version.

Reply:
good catch, I allready thought there should be a get_safe_patches, like some other get_safe, but now I know